proc_blocks/ohv_label/src/lib.rs

```diff
@@ -41,12 +46,20 @@ impl<const N: usize> OhvLabel<N> {
         OhvLabel {
             labels,
             unknown_label,
+            output_count: DEFAULT_OUTPUT,
```

Comment:
Should we keep using the previous `output_count` instead of silently resetting it to `DEFAULT_OUTPUT`?